### PR TITLE
Update file-filters.md

### PR DESCRIPTION
**Pull request by drormikdash**
clarification that the sensitivity label filter will cause File Policy to skip (no handle) non-Office files

### CODE REVIEW

**Comment by drormikdash** [comment]
#sign-off

File: CloudAppSecurityDocs/file-filters.md [hunk 1]
```diff
@@ -110,6 +110,10 @@ Below is a list of the file filters that can be applied. To provide you with a p
     - **Password encrypted** – Files whose content wasn't inspected because they're password protected by the user.
     - **Corrupt file** – Files whose content wasn't inspected because their contents couldn't be read.
 
+    >[!NOTE]
+    >
+    > - If this filter is used in File Policy - the Policy will apply to Microsoft Office files only and will ignore other file types.
```
Comment (ShannonLeavitt) [inline CloudAppSecurityDocs/file-filters.md:115]:
```suggestion
    > If this filter is used in a file policy, the policy will apply to Microsoft Office files only and will ignore other file types.
```

**Comment by ShannonLeavitt** [comment]
@batamig

IMPORTANT: When the changes are ready for publication, add a `#sign-off` comment to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Comment by batamig** [comment]
closing to handline in private repo

**Pull request closed without merging**